build/status/OF92aXovaW5vdXQvbGFrZXNfc3VtbWFyeS5mZWF0aGVyLmluZA.yml

```diff
@@ -1,14 +1,14 @@
 version: 0.3.0
 name: 8_viz/inout/lakes_summary.feather.ind
 type: file
-hash: 01fdee7ef26a23b2d67b4e479effe22d
-time: 2021-06-24 17:20:43 UTC
+hash: 6e023c226d873abf73c519e094de9f15
+time: 2021-06-29 22:45:41 UTC
 depends:
   7b_temp_merge/out/temp_data_with_sources.feather: 25da8bda2945dcf9f5712398ac5c599b
   7_config_merge/out/nml_lake_depth_values.rds.ind: 52db91dd6b76487f74f88f437dd53ac2
   2_crosswalk_munge/out/gnisname_nhdhr_xwalk.rds.ind: c651e323e93cdf3c8de1bd2e56c3eabc
   7_config_merge/out/nml_lat_lon_values.rds.ind: d6e0a77832bbbd658acb329a9b3620ff
-  7_config_merge/out/nml_H_A_values.rds.ind: a1ec8bdc5142623b210a43945beb9169
+  7_config_merge/out/nml_H_A_values.rds.ind: 27333cc8e2b6e529ab3a8a2ca60d49d1
```

Comment:
I think we should check that the Navico depths got into the new depth munging list, perhaps by setting a `browser()` within that function and making sure they are getting joined and added in. There is a chance that this update to H_A_ was triggered by the update of `iadnr_bathy` when you corrected that typo and the Navico data either aren't adding anything new or are somehow not getting included 🤔

Reply:
Ok - dropped in a `browser()` call and looks like the length of `use_ids` returned from the intersection of the Navico data and `still_empty_ids` is 392, so they should be getting added in.

Reply:
I've still got the local copy of `7_config_merge/out/nml_H_A_values.rds` prior to this build, so if you had one of the Navico `site_id`s that exists in your copy of that but would be missing from mine, I can check that. 

But we can deal with this later after the forecasting data things are figured out.

Reply:
Sounds good - I think it's not quite as simple as picking any id from the `use_ids` vector, since we're switching up the order in which depth data is added (e.g. Navico data used before other depth data). So I think there are at least some instances where Navico data should now be being used in place of other data that was used previously.

When I did the first point in polygon check, the Navico dataset only added 268 lakes, but when we provide the Navico data here as the 1st of the depth-only datasets, Navico provides data for 392 of the `still_empty_ids`. So for a given one of the 392 ids, we might already have depth data for that lake in your version of the file, it just might be from a source other than Navico. But the majority of those Navico-provided ids should truly be 'new' lakes.

Here are the first 18 of those 392 ids:
``` r
1] "nhdhr_83836501"                               "nhdhr_{2D616E50-968C-49A6-90A1-D860FF7B2408}" "nhdhr_{0BB28A37-D665-4735-A2DA-5C08014F9EEF}"
  [4] "nhdhr_85083962"                               "nhdhr_109986928"                              "nhdhr_85083224"                              
  [7] "nhdhr_109989898"                              "nhdhr_109987796"                              "nhdhr_154350325"                             
 [10] "nhdhr_156105000"                              "nhdhr_155494358"                              "nhdhr_90588342"                              
 [13] "nhdhr_{12C89B8D-1D97-4B56-96AC-463C0703B56C}" "nhdhr_123167703"                              "nhdhr_90569505"                              
 [16] "nhdhr_121624161"                              "nhdhr_121627248"                              "nhdhr_121624597"
```